extensions/azurecore/src/account-provider/auths/azureAuth.ts

```diff
@@ -358,7 +364,7 @@ export abstract class AzureAuth implements vscode.Disposable {
 
 			return tenants;
 		} catch (ex) {
-			Logger.log(ex);
+			Logger.write(LogLevel.Error, ex);
```

Comment:
Typically you should include text that indicates where the error happened as well (e.g. `Error fetching tenants : ${ex}`)